setup.py

```diff
@@ -306,7 +299,9 @@ def cmake(install_prefix: str = "./nvfuser"):
 
     logger.setLevel(logger_level)
 
-    pytorch_use_distributed = get_pytorch_use_distributed()
+    assert (
```

Comment:
@xwang233 is this the right way to check a misconfig? I usually use `assert` only for programming errors, but I'm fine with using it if no alternatives.

Reply:
I would personally prefer a `RuntimeError` clause like this one in apex https://github.com/NVIDIA/apex/blob/a7de60e57f0534266841e1733262601ad76aaa74/setup.py#L39-L47

Reply:
Done.